### DISCUSSION TITLE
Removing DRM from AZW3 file doesn't work, is that normal?

### DISCUSSION THREAD

**Discussion by leonfitz** [noDRM/DeDRM_tools#150] [Q&A]
Hi,
I installed the DeDRM 10.0.3 in Calibre 6.3. on MacOS Monterey 12.5.1, but removing DRM from an AZW3 Ebook didn't work, a popup appears which says it's DRM protected. Is it normal that AZW3 files don't work, or might it be another problem?
Thanks in advance!

**Comment by ElleKayEm** [comment]
AZW3 files work.  If the DRM is tied to an E-ink Kindle, enter the Kindle's serial number into DeDRM's customization.

**Reply by leonfitz** [reply]
Thanks! Already did that before. Now it worked. Do all AZW3 files work, also if they're not suitable for an E-ink kindle, because they have pictures and stuff?

**Reply by ElleKayEm** [reply]
You can also decrypt books from Kindle for PC/Mac.  See this post for details: https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by ElleKayEm** [reply]
I should say that post tells you how to get Amazon files that can be decrypted.  They don't actually allow discussion of DRM removal there.

**Reply by leonfitz** [reply]
> I should say that post tells you how to get Amazon files that can be decrypted. They don't actually allow discussion of DRM removal there.

Thanks for your help!
